styles.css

```diff
@@ -0,0 +1,542 @@
+
+* {
+  scrollbar-width: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none;
+}
+
+.check,
+.check * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+#header {
+  /* position: absolute; */
+  width: 100vw;
+  left: 0;
+  z-index: 10;
+  border-bottom: 2px solid gray;
+  background-color: white;
+}
+
+#header a {
+  color: black;
+  text-decoration: none;
+}
+
+#header a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#header-nav-bar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 37%;
+}
+
+.session-container {
+  display: none;
+}
+
+.secondary-nav {
+  display: none;
+}
+
+#menu {
+  position: absolute;
+  z-index: 1;
+}
+
+.close-menu-icon {
+  position: absolute;
+  display: none;
+  cursor: pointer;
+
+  /* opacity: 0; */
+}
+
+#menu-container {
+  position: absolute;
+  font-weight: bold;
+  top: 4em !important;
+  width: 100%;
+  height: 100%;
+
+  /* opacity: 0; */
+}
+
+.menu {
+  flex-direction: column;
+  align-items: center;
+  row-gap: 2rem;
+  padding: 1rem 1rem 1rem 1rem;
+  background-color: lightgray;
+  display: none;
+}
+
+.nav-li1:hover {
+  background-color: orangered;
+}
+
+#menu-icon {
+  cursor: pointer;
+  transition: all 0.3s ease-in-out;
+}
+
+#menu-icon:hover #menu-icon:focus {
+  transform: scale(1.1);
+  outline: none;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 2rem;
+  margin-top: 1.5rem;
+  width: 100%;
+}
+
+.hero-hello {
+  font-size: 25px;
+  color: black;
+}
+
+.hero-title {
+  font-size: 50px;
+  font-weight: 900;
+  color: orangered;
+  letter-spacing: -1.5px;
+  padding-bottom: 0;
+  background-image: url(../capstoneimgs/peakpx.jpg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-position: 60% 100%;
+}
+
+.artists-lineup {
+  justify-content: center;
+  width: 100%;
+  margin-top: 1vw;
+  margin-right: 16px;
+  background: url(../capstoneimgs/inti-raymi.png) center no-repeat;
+}
+
+.line1 {
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+.line2 {
+  font-size: 0.69em;
+  font-weight: 700;
+}
+
+.line3 {
+  font-size: 0.8rem;
+  font-weight: 700;
+}
+
+.line4 {
+  font-size: 0.6rem;
+  font-weight: 700;
+}
+
+.line5 {
+  font-size: 0.5rem;
+  font-weight: 700;
+}
+
+.line6 {
+  font-size: 1rem;
+  font-weight: 700;
+}
+
+.line7 {
+  font-size: 0.7rem;
+  font-weight: 700;
+}
+
+.line8 {
+  font-size: 0.7rem;
+  font-weight: 700;
```

Comment:
Kindly follow DRY principles and refactor your repeating values on your CSS code. You can create a class called .bold and give it as HTML attribute.
<img width="1108" alt="Screenshot 2023-03-18 at 12 46 43 PM" src="https://user-images.githubusercontent.com/25887253/226098107-174fa478-286b-4e9f-86f1-a62f341a9f1f.png">